docs/src/modules/components/overview/Customization.tsx

```diff
@@ -0,0 +1,397 @@
+    </SvgIcon>
+  );
+}
+function RoundedCornersIcon({ fontSize = 'medium' }: { fontSize: 'small' | 'medium' }) {
+  return (
+    <SvgIcon fontSize={fontSize} className="rounded" viewBox="0 0 20 20" fill="currentColor">
+      <path d="M15.6812 18C15.6812 10.4441 9.5559 4.31884 2 4.31884V2C10.8366 2 18 9.16344 18 18H15.6812Z" />
+    </SvgIcon>
+  );
+}
+
+const MD3Colors = {
+  default: '#6750A4',
+  pink: '#B54C9B',
+  green: '#588E5B',
+};
+const MD2Colors = {
+  default: '#1976d2',
+  purple: '#9c27b0',
+  pink: '#fe0265',
+};
+const customColors = {
+  default: '#5C5CF0',
+  orange: '#E08151',
+  green: '#306A5E',
+};
+
+const getColorScheme = (selectedTheme: Themes) => {
+  if (selectedTheme === 'md3') {
+    return MD3Colors;
+  }
+  if (selectedTheme === 'custom') {
+    return customColors;
+  }
+
+  return MD2Colors;
+};
+
+function ColorSwatch({ color }: { color: string }) {
+  return (
+    <Box
+      sx={(theme) => ({
+        width: '50px',
+        height: '12px',
+        background: color,
+        borderRadius: theme.shape.borderRadius,
+      })}
+    />
+  );
+}
+
+const getTheme = (mode: PaletteMode, config: Config, selectedTheme: Themes): ThemeOptions => {
+  if (selectedTheme === 'md3') {
+    return createTheme(getMD3Theme(mode, config));
+  }
+
+  if (selectedTheme === 'custom') {
+    return createTheme(getCustomTheme(mode, config));
+  }
+
+  return createTheme(getDefaultTheme(mode, config));
+};
+
+function CustomTheme({
+  selectedTheme,
+  mode = 'light',
+  config,
+}: {
+  selectedTheme: Themes;
+  mode: 'light' | 'dark';
+  config: Config;
+}) {
+  return (
+    <React.Fragment>
+      <CssBaseline />
+
+      <Stack sx={{ minHeight: '600px', justifyContent: 'center' }}>
+        <ThemeProvider theme={getTheme(mode, config, selectedTheme)}>
+          {selectedTheme === 'custom' || selectedTheme === 'default' ? (
+            <Card elevation={0} sx={{ padding: 0 }} variant="outlined">
+              <CustomLayoutRangePicker layout={config.layout} />
+            </Card>
+          ) : (
+            <Card elevation={0} sx={{ padding: 0 }}>
+              <StaticDatePicker
+                defaultValue={dayjs('2022-04-17')}
+                slotProps={{ layout: { sx: { minWidth: 'initial' } } }}
+              />
+            </Card>
+          )}
+        </ThemeProvider>
+      </Stack>
+    </React.Fragment>
+  );
+}
+
+const initialState: Config = {
+  selectedTheme: 'custom',
+  color: 'default',
+  layout: 'horizontal',
+  density: 'medium',
+  corner: 'medium',
+  typography: 'default',
+};
+
+export default function Customization() {
+  const [styleConfig, setStyleConfig] = React.useState<Config>(initialState);
+
+  const brandingTheme = useTheme();
+
+  const colorScheme = getColorScheme(styleConfig.selectedTheme);
+
+  const handleChangeTheme = (_event: React.MouseEvent<HTMLElement>, newTheme: Themes) => {
+    if (newTheme !== null) {
```

Comment:
Nit: I think it makes sense to do this for the other toggles too.

Reply:
I'll just leave this here: 🤦‍♀️